rasp/selenium_engine.py

```diff
@@ -1,91 +1,31 @@
 import selenium
 import selenium.webdriver
-import time
 
 from rasp.base import Engine, Webpage
 from rasp.errors import EngineError
 
 
 class BaseSeleniumEngine(Engine):
-    def __init__(self):
-        self.driver = None
-        return
+    def __init__(self, *args, **kwargs):
+        self.driver = selenium.webdriver.Firefox()
```

Comment:
I'd in general agree, set-up-like stuff should go in __init__, and it'd be cool to have enter/exit populated to call setup/cleanup (and init call setup) to allow those to be used as custom behavior.  I'm not sure I have a great handle on the actual mechanism for a user implementing that custom behavior yet, but that'd be out of scope for this PR anyway, so we can flesh that out separately later on.